crates/polars-ops/src/series/ops/approx_unique.rs

```diff
@@ -6,7 +6,7 @@ use polars_core::with_match_physical_integer_polars_type;
 #[cfg(feature = "approx_unique")]
 use crate::series::HyperLogLog;
 
-fn approx_unique_ca<'a, T>(ca: &'a ChunkedArray<T>) -> PolarsResult<Series>
```

Comment:
We should probably rename this file as well to `approx_n_unique.rs`

Reply:
I was wondering about that; one thought - are other any other related algorithms likely to be added (for instance, an _actual_ `approx_unique`?) If so would make sense to keep them in the same place. Though we can always rename now, and simply rename again _if_ that ever happens  ;)

Reply:
> Though we can always rename now, and simply rename again if that ever happens ;)

That would generally be my approach. Though if you prefer leaving it as-is, that's also OK with me :)

Reply:
As for the name, a quick bit of googling makes me think that most packages that support approximate functions lead with `approx_` (probably to naturally cluster them together in docs/lookups/etc?)

For instance: BigQuery [approx funcs](https://cloud.google.com/bigquery/docs/reference/standard-sql/approximate_aggregate_functions).
Snowflake leads with the algorithm, prefixing their approximate functions with `HLL_`; let's definitely _not_ do that :)

Reply:
Let's keep it like this then!